hello-strongbox-pypi/setup.py

```diff
@@ -4,7 +4,7 @@
     long_description = fh.read()
 
 setup(
```

Comment:
From [Python Packaging docs](https://packaging.python.org/tutorials/packaging-projects/):

> name is the distribution name of your package. This can be any name as long as only contains letters, numbers, _ , and -. It also must not already be taken on pypi.org. Be sure to update this with your username, as this ensures you won’t try to upload a package with the same name as one which already exists when you upload the package.

> packages is a list of all Python import packages that should be included in the Distribution Package.

Reply:
Thanks for this clarification @j4ckofalltrades .

cc: @steve-todorov , @anki2189

Reply:
Thanks for pinging me! 

cc: @sbespalov in case you don't already know this. :)